libs/coin-modules/coin-bitcoin/src/wallet-btc/__tests__/xpub.syncing.integration.test.ts

```diff
@@ -126,14 +126,6 @@ describe("xpub integration sync", () => {
       network: coininfo.qtum.main.toBitcoinJS(),
       currencyId: "qtum",
     },
-    {
```

Comment:
I remove this for peercoin as there is a 500 network error related to our peercoin endpoint and this test is broken and we don't want to support peercoin now.